code/go/0chain.net/blobbercore/allocation/allocationchange.go

```diff
@@ -39,6 +39,21 @@ type AllocationChangeProcessor interface {
 	Unmarshal(string) error
 }
 
+type IAllocationChangeCollector interface {
+	TableName() string
+	AddChange(allocationChange *AllocationChange, changeProcessor AllocationChangeProcessor)
+	Save(ctx context.Context) error
```

Comment:
A simpler solution would be just to create a function and put save and addchange inside the function. You can then mock that function instead of this

Reply:
I can change this and add a function like AddChangeAndSave. But AddChange and Save functions were already there. I simply added them to the interface

Reply:
Git diff said you created this interface - if you only added 2 functions it would show only those